keps/sig-apps/4368-support-managed-by-for-batch-jobs/README.md

```diff
@@ -974,6 +1075,10 @@ N/A.
 
 - 2023-12.20 - First version of the KEP
 - 2024-03-05 - Merged implementation PR [Support for the Job managedBy field (alpha)](https://github.com/kubernetes/kubernetes/pull/123273)
+- 2024-03-07 - Merged [Update Job conformance test for job status updates](https://github.com/kubernetes/kubernetes/pull/123751)
```

Comment:
non-blocking PRR things:

> Are there any tests for feature enablement/disablement?

Were the tests added? If not, can we prioritize them for the second Alpha?

Reply:
Yes, we have the unit tests for enablement of the feature: https://github.com/kubernetes/kubernetes/blob/169a952720ebd75fcbcb4f3f5cc64e82fdd3ec45/pkg/controller/job/job_controller_test.go#L2365

We also have an integration test for re-enabling the feature: https://github.com/kubernetes/kubernetes/blob/169a952720ebd75fcbcb4f3f5cc64e82fdd3ec45/test/integration/job/job_test.go#L1691.

Reply:
> Yes, we have the unit tests for enablement of the feature: https://github.com/kubernetes/kubernetes/blob/169a952720ebd75fcbcb4f3f5cc64e82fdd3ec45/pkg/controller/job/job_controller_test.go#L2365

These aren't enablement tests - these are feature tests.

Please take a look at KEP template - I'm looking for simple tests that change the value of FG in the middle of the tests.

Reply:
> Please take a look at KEP template - I'm looking for simple tests that change the value of FG in the middle of the tests.

I think the integration test satisfies this expectation.  See the flips in the middle [disabling](https://github.com/kubernetes/kubernetes/blob/169a952720ebd75fcbcb4f3f5cc64e82fdd3ec45/test/integration/job/job_test.go#L1745), [re-enabling](https://github.com/kubernetes/kubernetes/blob/169a952720ebd75fcbcb4f3f5cc64e82fdd3ec45/test/integration/job/job_test.go#L1763)

I don't think we have a unit test like that, do you think we should also add unit, given we have integration?

Reply:
missed the integration, sorry. This is fine.

Just please link it here in this question explicitly.

Reply:
Sure, updated with the link